src/find-options/operator/Raw.ts

```diff
@@ -4,6 +4,6 @@ import {FindOperator} from "../FindOperator";
  * Find Options Operator.
  * Example: { someField: Raw([...]) }
  */
-export function Raw<T>(value: string|((columnAlias?: string) => string)) {
-    return new FindOperator("raw", value as any, false);
+export function Raw<T>(value: string|((columnAlias?: string) => string)): any { // TODO: make this typecheck
```

Comment:
why did you change it? There are no compiler errors we currently have.

Reply:
strange.. I'm running into a compiler warning saying T is unused. I can revert this if this error is ignored.

Reply:
make sure you use correct typescript version. There are no typescript errors in CI and my local machine